src/Channel.ts

```diff
@@ -244,6 +279,14 @@ export default class Channel {
     this.setLut(lut.lut);
   }
 
+  public lutGenerator_minMax(imin: number, imax: number): void {
+    if (!this.loaded) {
+      return;
+    }
+    const lut = this.histogram.lutGenerator_minMax(imin, imax);
+    this.setLut(lut.lut);
+  }
```

Comment:
this function was added as a convenience function for when we want to create a lut just from a min and a max value. (Equivalent to the window/level function but a different formulation of the inputs)